cql3/restrictions/statement_restrictions.cc

```diff
@@ -345,7 +345,7 @@ static value_set possible_lhs_values(const column_definition* cdef,
                             if (cdef) {
                                 return unbounded_value_set;
                             }
-                            const auto val = evaluate(oper.rhs, options).to_managed_bytes_opt();
+                            auto val = evaluate(oper.rhs, options).to_managed_bytes_opt();
                             if (!val) {
                                 return empty_value_set; // All NULL comparisons fail; no token values match.
                             }
```

Comment:
nit: Here below there's

```c++
                            if (oper.op == oper_t::EQ) {
                                return value_list{*val};
```

code that's worth patching like

```patch
                             if (oper.op == oper_t::EQ) {
-                                return value_list{*val};
+                                return value_list{std::move(*val)};
```

(incrementally, of course)